src/test/java/org/ieeervce/api/siterearnouveau/service/EventsServiceTest.java

```diff
@@ -1,32 +1,37 @@
 package org.ieeervce.api.siterearnouveau.service;
 
-import static org.assertj.core.api.Assertions.assertThat;
-import static org.mockito.Mockito.when;
-
-import java.util.Collections;
-
 import org.ieeervce.api.siterearnouveau.entity.Event;
 import org.ieeervce.api.siterearnouveau.repository.EventsRepository;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
 import org.mockito.InjectMocks;
 import org.mockito.Mock;
 import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.data.domain.*;
+
+import java.util.Collections;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.when;
 
 @ExtendWith(MockitoExtension.class)
 class EventsServiceTest {
     @Mock
     EventsRepository eventsRepository;
     @InjectMocks
-    EventsService eventsService;
+    EventsService    eventsService;
 
     @Mock
     Event event;
 
     @Test
-    void testList(){
-        when(eventsRepository.findAll()).thenReturn(Collections.singletonList(event));
-        var eventsList = eventsService.list();
-        assertThat(eventsList).isNotNull().singleElement().isSameAs(event);
+    void testList() {
+        Pageable pageable = Pageable.ofSize(20).withPage(0);
```

Comment:
Can you use default Page size? instead of 20?

Reply:
Default is 20